modules/preparing-an-inital-cluster-deployment-for-mce-disconnected.adoc

```diff
@@ -0,0 +1,131 @@
+// Module included in the following assemblies:
+//
+// * installing_with_agent_based_installer/preparing-an-agent-based-installed-cluster-for-mce.adoc
+
+:_content-type: PROCEDURE
+[id="preparing-an-inital-cluster-deployment-for-mce-disconnected_{context}"]
+
+= Preparing an agent-based cluster deployment for multicluster engine for Kubernetes (MCE) Operator while disconnected
+
+You can mirror the required {product-title} container images, the multicluster engine for Kubernetes (MCE) Operator, and Local Storage Operator (LSO) into your local mirror registry in a disconnected environment.
+Ensure that you note the local DNS hostname and port of your mirror registry.
+
+[NOTE]
+====
+To mirror your {product-title} image repository to your mirror registry, you can use either the `oc adm release image`
+or `oc mirror` command. In this procedure, the `oc mirror` command is used as an example.
+====
+
+.Procedure
+
+. Create an `<assets_directory>` folder to contain valid `install-config.yaml` and `agent-config.yaml` files. This directory is used to store all the assets.
+
+. To mirror a {product-title} image repository, the MCE Operator, and Local Storage Operator(LSO), create a `ImageSetConfiguration.yaml` file with the following settings:
+
++
+.Example `ImageSetConfiguration.yaml`
+
+[source,yaml]
+----
+  kind: ImageSetConfiguration
+  apiVersion: mirror.openshift.io/v1alpha2
+  archiveSize: 4 <1>
+  storageConfig: <2>
+    imageURL: <your-local-registry-dns-name>:<your-local-registry-port>/mirror/oc-mirror-metadata <3>
+    skipTLS: true
+  mirror:
+    platform:
+      architectures:
+        - "amd64"
+      channels:
+        - name: stable-4.12 <4>
+          type: ocp
+    additionalImages:
+      - name: registry.redhat.io/ubi8/ubi:latest
+    operators:
+      - catalog: registry.redhat.io/redhat/redhat-operator-index:v4.12 <5>
+        packages: <6>
+          - name: multicluster-engine <7>
+          - name: local-storage-operator <8>
+----
++
+<1> Set the maximum size, in GiB, of each file within the image set.
+<2> Set the back-end location to receive the image set metadata. This location can be a registry or local directory. It is required to specify `storageConfig` values, unless you are using the Technology Preview OCI feature.
+<3> Set the registry URL for the storage backend.
+<4> Set the channel that contains the {product-title} images for the version you are installing.
+<5> Set the Operator catalog that contains the {product-title} images that you are installing.
+<6> Specify only certain Operator packages and channels to include in the image set. Remove this field to retrieve all packages in the catalog.
+<7> The MCE Operator packages and channels.
+<8> The LSO Operator packages and channels.
+
```

Comment:
Everywhere in here where there is a blank line before a line with a `+`, the blank line should be deleted (there are a few instances).
```suggestion
```

Reply:
Addressed in both mce modules